config/crd/kustomization.yaml

```diff
@@ -1,39 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-
-# This kustomization.yaml is not intended to be run by itself,
-# since it depends on service name and namespace that are out of this kustomize package.
-# It should be run by config/default
 resources:
-- bases/gateway-operator.konghq.com_controlplanes.yaml
-- bases/gateway-operator.konghq.com_dataplanes.yaml
-- bases/gateway-operator.konghq.com_gatewayconfigurations.yaml
-- bases/gateway-operator.konghq.com_dataplanemetricsextensions.yaml
-- bases/gateway-operator.konghq.com_kongplugininstallations.yaml
-- bases/gateway-operator.konghq.com_konnectextensions.yaml
-#+kubebuilder:scaffold:crdkustomizeresource
-
-# patches:
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix
-# and ensure they are enclosed under patches key above.
-# patches here are for enabling the conversion webhook for each CRD
-#- path: patches/webhook_in_controlplanes.yaml
-#- path: patches/webhook_in_dataplanes.yaml
-#- path: patches/webhook_in_gatewayconfigurations.yaml
-#- patches/webhook_in_dataplanes.yaml
-#- patches/webhook_in_aigateways.yaml
-#+kubebuilder:scaffold:crdkustomizewebhookpatch
-
-# [CERTMANAGER] To enable cert-manager, uncomment all the sections with [CERTMANAGER] prefix
-# and ensure they are enclosed under patches key above.
-# patches here are for enabling the CA injection for each CRD
-#- path: patches/cainjection_in_controlplanes.yaml
-#- path: patches/cainjection_in_dataplanes.yaml
-#- path: patches/cainjection_in_gatewayconfigurations.yaml
-#- patches/cainjection_in_dataplanes.yaml
-#- patches/cainjection_in_aigateways.yaml
-#+kubebuilder:scaffold:crdkustomizecainjectionpatch
-
-# the following config is for teaching kustomize how to do kustomization for CRDs.
-configurations:
-- kustomizeconfig.yaml
+  - https://github.com/kong/kubernetes-configuration/config/crd/gateway-operator?ref=main
```

Comment:
This should use the version that's defined in `go.mod` in https://github.com/Kong/gateway-operator/blob/887cc89d8559bd8fe52e94f029deb0e0c4f65139/go.mod#L22

We can leverage the script that KIC has in https://github.com/Kong/kubernetes-ingress-controller/blob/03fe69e268019d2dd0018615976d89f435b60743/scripts/generate-crd-kustomize.sh to generate the kustomization files (e.g. this one https://github.com/Kong/kubernetes-ingress-controller/blob/b852f2d18640efc9a68589168c63a59a9f7cb6ac/config/crd/kustomization.yaml)